### PR TITLE
Tailwind CSS preparation for the project is done

### PR DESCRIPTION
**Pull request by RashEstebek**
- added three folders to the public folder for the images
- added index.css file for the styles
- added all the necessary tailwind configuration files
- added custom variables for the our project's unique colors

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,8 +9,15 @@
     "preview": "vite preview"
   },
   "devDependencies": {
+    "autoprefixer": "^10.4.16",
+    "postcss": "^8.4.32",
```
Comment (baglana) [inline package.json:13]: what are these for?

**Review by baglana — APPROVED** [review]
Good!

File: package.json (hunk 1 above)
Comment (RashEstebek) [reply package.json:13]: I don't know, I followed instructions for the tailwind installation in vite. And these two dependencies also installed. I suppose, these dependencies are for tailwind to work properly

File: src/main.ts [hunk 2]
```diff
@@ -1,24 +1,3 @@
...
-    <p class="read-the-docs">
-      Click on the Vite and TypeScript logos to learn more
-    </p>
-  </div>
-`
-
-setupCounter(document.querySelector<HTMLButtonElement>('#counter')!)
+import './index.css'
+import 'reset-css'
+import '@fontsource-variable/plus-jakarta-sans';
```
Comment (talgat-ruby) [inline src/main.ts]: I think the order matter. Add lib styles first, all other css last.

**Review by talgat-ruby — CHANGES_REQUESTED** [review]

File: src/main.ts (hunk 2 above)
Comment (RashEstebek) [reply src/main.ts]: ok

Comment (RashEstebek) [reply src/main.ts]: I changed. Are the changes visible?

Comment (talgat-ruby) [reply src/main.ts]: yes

**Review by talgat-ruby — APPROVED** [review]

**Pull request merged**